[
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 08:00:03,896 INFO New Trending 2019-08-13\n\n2019-08-13 08:00:03,902 INFO daily.py took 0:00:00.692566\n```",
        "ts": "1565683178.003900",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 1.063s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565683491.004000",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 08:13:21,122 INFO New Trending 2019-08-13\n{\"name\": \"Talisman of Indulgence\", \"s\": [0, 28, 20, 26, 61, 55, 53], \"z\": 1.189272915818414}\n{\"name\": \"Squee, Goblin Nabob\", \"s\": [0, 8, 1, 11, 29, 27, 23], \"z\": 1.1661294358105199}\n{\"name\": \"Animate Dead\", \"s\": [0, 40, 45, 45, 79, 85, 77], \"z\": 1.1577696434777984}\n{\"name\": \"Drownyard Temple\", \"s\": [0, 6, 7, 9, 20, 19, 21], \"z\": 1.1535063642433385}\n{\"name\": \"Bone Miser\", \"s\": [0, 0, 0, 8, 24, 17, 22], \"z\": 1.1505684271808827}\n{\"name\": \"Akoum Refuge\", \"s\": [0, 15, 9, 19, 35, 31, 30], \"z\": 1.1346847777389153}\n{\"name\": \"Night's Whisper\", \"s\": [0, 23, 21, 28, 43, 47, 45], \"z\": 1.1236375235986378}\n{\"name\": \"Call to the Netherworld\", \"s\": [0, 2, 3, 10, 17, 16, 16], \"z\": 1.1215784033582528}\n{\"name\": \"Doomed Necromancer\", \"s\": [0, 2, 3, 9, 22, 17, 15], \"z\": 1.1209630159523867}\n{\"name\": \"Dread Return\", \"s\": [0, 14, 12, 16, 32, 27, 28], \"z\": 1.1201804473157375}\n2019-08-13 08:13:21,138 INFO daily.py took 0:13:16.288824\n```",
        "ts": "1565683990.004100",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 08:13:51,137 INFO backup.py took 0:13:49.302693\n```",
        "ts": "1565684020.004200",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 0.905s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565690690.004300",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 11:54:39,671 CRITICAL Uncaught exception\nTraceback (most recent call last):\n  File \"&lt;stdin&gt;\", line 1, in &lt;module&gt;\n  File \"\/home\/ubuntu\/edhrec\/web\/nightly.py\", line 470, in make_precons\n    ) for name, facecard in sorted(precons.items(), key=lambda x: x[0])]\n  File \"\/home\/ubuntu\/edhrec\/web\/nightly.py\", line 470, in &lt;listcomp&gt;\n    ) for name, facecard in sorted(precons.items(), key=lambda x: x[0])]\nTypeError: Can't convert 'bool' object to str implicitly\n```",
        "ts": "1565697268.004400",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 11:57:04,489 CRITICAL Uncaught exception\nTraceback (most recent call last):\n  File \"&lt;stdin&gt;\", line 1, in &lt;module&gt;\n  File \"\/home\/ubuntu\/edhrec\/web\/nightly.py\", line 470, in make_precons\n    ) for name, facecard in sorted(precons.items(), key=lambda x: x[0])]\n  File \"\/home\/ubuntu\/edhrec\/web\/nightly.py\", line 470, in &lt;listcomp&gt;\n    ) for name, facecard in sorted(precons.items(), key=lambda x: x[0])]\nTypeError: Can't convert 'bool' object to str implicitly\n```",
        "ts": "1565697413.004500",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 11:57:59,216 CRITICAL Uncaught exception\n  File \"&lt;stdin&gt;\", line 1\n    pdict['Faceless Menace'0\n                           ^\nSyntaxError: invalid syntax\n```",
        "ts": "1565697468.004600",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 12:01:10,482 CRITICAL Uncaught exception\nTraceback (most recent call last):\n  File \"&lt;stdin&gt;\", line 1, in &lt;module&gt;\nTypeError: 'builtin_function_or_method' object is not iterable\n```",
        "ts": "1565697659.004900",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 12:03:44,026 INFO cards.py took 0:03:38.984099\n```",
        "ts": "1565697812.005000",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "Nightly started, onlydecks=False, skipdecks=False",
        "ts": "1565697820.005100",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 9.727s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565697899.005200",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 12:11:40,152 CRITICAL Uncaught exception\nTraceback (most recent call last):\n  File \"&lt;stdin&gt;\", line 1, in &lt;module&gt;\nTypeError: 'module' object is not callable\n```",
        "ts": "1565698289.005300",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 12:32:56,572 CRITICAL Uncaught exception\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/core\/cards.py\", line 485, in get\n    return _CARDS[sanitize_url(name)]\nKeyError: 'snow-covered-swamp'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"&lt;stdin&gt;\", line 1, in &lt;module&gt;\n  File \"\/home\/ubuntu\/edhrec\/core\/cards.py\", line 487, in get\n    return _CARDS['chaos-orb']\nKeyError: 'chaos-orb'\n```",
        "ts": "1565699565.005400",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 12:33:31,461 CRITICAL Uncaught exception\nTraceback (most recent call last):\n  File \"&lt;stdin&gt;\", line 1, in &lt;module&gt;\nNameError: name 'cache' is not defined\n```",
        "ts": "1565699600.005500",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 12:34:09,166 CRITICAL Uncaught exception\nTraceback (most recent call last):\n  File \"&lt;stdin&gt;\", line 1, in &lt;module&gt;\nImportError: No module named 'core'\n```",
        "ts": "1565699638.005600",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 1.922s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565705092.000100",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "Nightly completed",
        "ts": "1565711394.000200",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 15:50:06,542 INFO nightly.py took 3:46:14.015295\n```",
        "ts": "1565711395.000300",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 26.885s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565712316.000400",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 9.456s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565719498.000500",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 1.160s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565726690.000700",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 20:29:50,913 INFO archidekt.py took 0:25:46.639765\n```",
        "ts": "1565728179.000800",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 20:57:44,720 INFO deckstats.py took 0:25:52.486592\n```",
        "ts": "1565729853.000900",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 21:21:36,891 CRITICAL Uncaught exception\n  File \"&lt;stdin&gt;\", line 1\n    cd ..\n        ^\nSyntaxError: invalid syntax\n```",
        "ts": "1565731285.001000",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 21:46:19,433 INFO mtggoldfish.py took 0:28:18.170553\n```",
        "ts": "1565732767.001200",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 1.674s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565733890.001300",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 22:31:52,487 INFO update_deckswithprices.py took 0:29:22.240672\nNew deck count from past 2 days: \n- deckstats: 532\n- Other: 0\n- MTGGoldfish: 596\n- Archidekt: 778\n```",
        "ts": "1565735500.001400",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 22:35:22,146 ERROR Dynamic too many open files - restarting\n2019-08-13 22:35:22,435 ERROR Dynamic too many open files - restarting\n```",
        "ts": "1565735710.001500",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-13 22:35:23,185 CRITICAL Uncaught exception\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/core\/cache.py\", line 23, in _getcursor\n    yield con.cursor(cursor_factory=psycopg2.extras.DictCursor)\n  File \"\/home\/ubuntu\/edhrec\/core\/cache.py\", line 89, in set\n    (key, value, expires)\n  File \"\/usr\/local\/lib\/python3.5\/dist-packages\/psycopg2\/extras.py\", line 141, in execute\n    return super(DictCursor, self).execute(query, vars)\npsycopg2.OperationalError: terminating connection due to administrator command\nserver closed the connection unexpectedly\n\tThis probably means the server terminated abnormally\n\tbefore or while processing the request.\n\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/core\/wordpress.py\", line 295, in &lt;module&gt;\n    update_all()\n  File \"\/home\/ubuntu\/edhrec\/core\/wordpress.py\", line 282, in update_all\n    _get_multipage('posts')\n  File \"\/home\/ubuntu\/edhrec\/core\/wordpress.py\", line 56, in _get_multipage\n    cache.set('wp\/{}'.format(endpoint), out)\n  File \"\/home\/ubuntu\/edhrec\/core\/cache.py\", line 89, in set\n    (key, value, expires)\n  File \"\/usr\/lib\/python3.5\/contextlib.py\", line 77, in __exit__\n    self.gen.throw(type, value, traceback)\n  File \"\/home\/ubuntu\/edhrec\/core\/cache.py\", line 25, in _getcursor\n    con.commit()\npsycopg2.InterfaceError: connection already closed\n```",
        "ts": "1565735711.001600",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "client_msg_id": "3e101ca0-9952-40f5-b25e-a795b78599de",
        "type": "message",
        "text": ".",
        "user": "U28TLDP0R",
        "ts": "1565735715.001800",
        "team": "T286HM2F3",
        "user_team": "T286HM2F3",
        "source_team": "T286HM2F3",
        "user_profile": {
            "avatar_hash": "2a61603e280e",
            "image_72": "https:\/\/avatars.slack-edge.com\/2016-11-16\/104915990673_2a61603e280ecee0ab38_72.jpg",
            "first_name": "David",
            "real_name": "David Keatts",
            "display_name": "keattz",
            "team": "T286HM2F3",
            "name": "keattz",
            "is_restricted": false,
            "is_ultra_restricted": false
        }
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\nF.............................\n======================================================================\nFAIL: test_Atarka (__main__.testXenagos)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File \"\/home\/ubuntu\/edhrec\/batch\/testing.py\", line 17, in test_Atarka\n    self.assertNotEqual(-1, middle.find('highsynergycards'.encode()))\nAssertionError: -1 == -1\n\n----------------------------------------------------------------------\nRan 30 tests in 1.560s\n\nFAILED (failures=1)\n\n```",
        "ts": "1565741090.001900",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "```2019-08-14 01:19:15,482 INFO recommendations.py took 17:19:10.871304\n```",
        "ts": "1565745543.002000",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 1: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 2: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 3: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 5: \/home\/ubuntu\/edhrec\/batch\/testing.py: Syntax error: \"(\" unexpected\n\n```",
        "ts": "1565748289.002100",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 1: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 2: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 3: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 5: \/home\/ubuntu\/edhrec\/batch\/testing.py: Syntax error: \"(\" unexpected\n\n```",
        "ts": "1565755489.002200",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    },
    {
        "type": "message",
        "subtype": "bot_message",
        "text": "\n```\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 1: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 2: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 3: \/home\/ubuntu\/edhrec\/batch\/testing.py: import: not found\n\/home\/ubuntu\/edhrec\/batch\/testing.py: 5: \/home\/ubuntu\/edhrec\/batch\/testing.py: Syntax error: \"(\" unexpected\n\n```",
        "ts": "1565762688.000100",
        "username": "edhrecbot",
        "icons": {
            "emoji": ":robot_face:",
            "image_64": "https:\/\/a.slack-edge.com\/37d58\/img\/emoji_2017_12_06\/apple\/1f916.png"
        },
        "bot_id": "B2867AJ5C"
    }
]